src/Pmi/WorkQueue/WorkQueue.php

```diff
@@ -340,7 +340,8 @@ public function generateTableRows($participants, $app)
             $row['dvEhrStatus'] = $this->displayConsentStatus($participant->consentForDvElectronicHealthRecordsSharing, $participant->consentForDvElectronicHealthRecordsSharingAuthored);
             $row['caborConsent'] = $this->displayConsentStatus($participant->consentForCABoR, $participant->consentForCABoRAuthored);
             $row['activityStatus'] = $this->getActivityStatus($participant);
-            $row['withdrawalStatus'] = $participant->withdrawalStatus; // Used to add withdrawn class in the data tables
+            $row['withdrawalStatus'] = $participant->withdrawalStatus;
```

Comment:
This can _likely_ be dropped from the payload if the prior comment is correct that it is only used for formatting.

Reply:
I agree that this can be removed. The only place it was being used was in the `createdRow` callback in workqueue.js which you've already modified to use the new field.